libmscore/chord.cpp

```diff
@@ -3098,7 +3098,36 @@ void Chord::sortNotes()
       std::sort(notes().begin(), notes().end(),
          [](const Note* a,const Note* b)->bool { return b->line() < a->line(); }
          );
-     }
+      }
+
+//---------------------------------------------------------
+//   nextTiedChord
+//    Return next chord if all notes in this chord are tied to it.
+//    Note: the next chord might have extra notes that are not tied
+//    back to this one. Set sameSize=true to return 0 in this case.
+//---------------------------------------------------------
+
+Chord* Chord::nextTiedChord(bool backwards, bool sameSize)
```

Comment:
Done. It doesn't show up in the preview on the comment page for some reason, but [it's there](https://github.com/musescore/MuseScore/pull/2784/files#diff-428e0bb12d899d2d494b3cc99e75e22fR3106).